integrations/amplitude/lib/index.js

```diff
@@ -464,3 +485,16 @@ function mapRevenueAttributes(track) {
     revenue: track.revenue()
   };
 }
+
+// map product for Order Completed event
+// REF: https://segment.com/docs/destinations/amplitude/#order-completed
+function mapProductAttributes(product) {
```

Comment:
Should we map falsy values or remove them?

Reply:
👆 Wondering if this should be `return reject({....})`?

Reply:
I would prefer not to use a dependency that has been moved to boneyard. We don't use reject in other parts of this destination, so seems ok to leave it as it is.